data/cookbook/text-processing/utf8-text-processing/00-camomile.ml

```diff
@@ -0,0 +1,64 @@
+---
+packages:
+- name: "camomile"
+  version: "1.0.2"
+libraries:
+- camomile
+discussion: |
+  - **Understanding `camomile`:** The `camomile` package provides functions which handles correcly UTF-8 characters.
+---
+
+(* Initialize the `camomile` library, which offer a large set of functions
+   to deal with strings presented in various encoding. The `Camomile`
+   module declaration is required with the version 1 of `camomile`
+   and mustn't be declared with the version 2: *)
+module Camomile = CamomileLibrary.Make(CamomileDefaultConfig)
+
+(* Get the length (number of characters) of a UTF-8 string: *)
+let () = assert (Camomile.UTF8.length "déjà" = 4)
+
+(* Validate a UTF-8 string. Would raise a `CamomileLibrary__UTF8.Malformed_code`
+   with a string like "\233\233" *)
+let () = assert (Camomile.UTF8.validate "déjà" = ())
+
+(* Extract the nth character from a string (the first character has a 0 index).
+   The character has a `Camomile.UChar.uchar` type which can represent
+   any Unicode character: *)
+let () = assert (Camomile.UChar.code (Camomile.UTF8.get "déjà" 3)
+                    = 224)
+
+(* Using bytes oriented indexes. Using functions which works
+   directly with the byte indexes of characters can be more efficient
+   than using functions like `get` or `length` that have to parse
+   the string counting characaters. `first`, `next`, `prev`, `last`
+   and `look` all deals with bytes indexes. `nth` convert of position
+   expressed as a character index to a byte index. *)
```

Comment:
Should `nth` be in monospace like it is here? If so, does it also need to be in monospace in li 24?

This sentence doesn't make sense. I suspect there's a typo.
"`nth` convert of position expressed as a character index to a byte index."

Should it be "`nth` converts a position expressed as a character index to a byte index" ?